bin/ghe-backup

```diff
@@ -189,8 +189,12 @@ ghe-backup-es-hookshot || failures="$failures hookshot"
 echo "Backing up Git repositories ..."
 ghe-backup-repositories || failures="$failures repositories"
 
-echo "Backing up GitHub Pages ..."
-ghe-backup-pages || failures="$failures pages"
+if [ "$GHE_RESTORE_SKIP_PAGES" = "yes" ]; then
+  echo "Skipping backup of GitHub Pages artifacts ..."
```

Comment:
Looks like we added the word 'artifacts' here compared to before. Intentional?

```suggestion
  echo "Skipping backup of GitHub Pages ..."
```

Reply:
Intentional, yes as there's a distinction between pages data and pages artifacts.
I'm not married to the term artifact but we're using it to distinguish between the generated byproducts and the source data that is always backed up.

Reply:
Good thinking